YoCode/FileImport.cs

```diff
@@ -13,17 +13,22 @@ namespace YoCode
 
     public class FileImport
     {
-        public string ORIGINAL_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\junior-test";
-        public string MODIFIED_PATH { get; set; } = @"C:\Users\ukmzil\source\repos\original-test";
+        // This shouldn't live in FileImport class
+
+        public string ORIGINAL_PATH { get; set; } = @"..\..\..\..\..\junior-test";
+
+        public string MODIFIED_PATH { get; set; } = @"..\..\..\..\..\original-test";
```

Comment:
I still think these paths need a better solution. I think we need to make these come from command-line arguments, so I'll ignore it for now until we get the command-line stuff working. Also ALLCAPS_CASE isn't normal for properties, TitleCase would be better. But we can fix that later too.